server/src/db/migrations/20220203095941_createVotes.cjs

```diff
@@ -0,0 +1,24 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.createTable("votes", (table) => {
+    table.bigIncrements("id")
+    table.bigInteger("userId").notNullable().references("users.id")
+    table.bigInteger("characterId").notNullable().references("characters.id")
+    table.integer("voteValue").notNullable()
```

Comment:
probably want to name this `value` so there isn't duplication in table/column name, is a little odd to call `vote.voteValue`, as opposed to `vote.value`

Reply:
this still needs to be updated